Makefile

```diff
@@ -30,6 +30,11 @@ endif
 
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./.git/*")
 
+VERSION?=latest
+ifeq ($(VERSION),latest)
+VERSION=$(shell curl -s https://api.github.com/repos/hortonworks/cloud-haunter/releases/latest | jq -r '.name')
```

Comment:
This line would fail if someone doesn't have `jq`. I suggest solving this before merging.

Reply:
+1. Also, it will run every time you do not specify a VERSION, so I think it should be moved under the download target.

Reply:
Now it is jq free.